workflow-core/README.md

```diff
@@ -2,6 +2,13 @@
 
 This module contains the core workflow APIs.
 
+## Kotlin Multiplatform
+
+This module is a Kotlin Multiplatform module. The targets currently included for build and test
+are `jvm`, `ios`, and `iosSimulatorSimulatorArm64`. If you are having issues with the tests,
```

Comment:
`iosSimulatorSimulatorArm64` is this a typo? Also, I thought we added js support too.